src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,60 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY_CONS = "supply";
+    private static final String BUY_CONS = "buy";
+    private static final String RESULT_CONS = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] totals = readFile(fromFileName);
+        int totalSupply = totals[0];
+        int totalBuy = totals[1];
+        writeToFile(toFileName, totalSupply, totalBuy);
+    }
+
+    public int[] readFile(String fromFileName) {
```

Comment:
The `readFile` method should be private if it is only used within this class. This will adhere to the encapsulation principle.